core/src/main/java/org/apache/accumulo/core/util/OpTimer.java

```diff
@@ -20,17 +20,20 @@
 
 import static java.util.concurrent.TimeUnit.NANOSECONDS;
 
+import java.time.Duration;
 import java.util.concurrent.TimeUnit;
 
+import org.apache.accumulo.core.util.time.NanoTime;
+
 /**
  * Provides a stop watch for timing a single type of event. This code is based on the
  * org.apache.hadoop.util.StopWatch available in hadoop 2.7.0
  */
 public class OpTimer {
 
   private boolean isStarted;
-  private long startNanos;
-  private long currentElapsedNanos;
+  private NanoTime start;
+  private Duration currentElapsed = Duration.ZERO;
```

Comment:
Rather than change the implementation, it looks like this is another class that tries to do the same kinds of things as the new NanoTime class. Their functions should be merged, and we should have only one, not both. Instead of changing this class, it'd be better to see if its uses can be replaced so it can be deleted.

Related: the NanoTime class doesn't actually expose the nanosecond units outside of the class. So, it doesn't need to be named "NanoTime" at all. It could just be called "OpTimer" and replace this class, or called "Stopwatch" or "ElapsedTime" or "Timer" or something else.